ibis/expr/api.py

```diff
@@ -330,13 +335,45 @@ def table(
         A schema for the table
     name
         Name for the table. One is generated if this value is `None`.
+    data
+        Table literal data.
+    columns
+        A sequence of [`str`][str] column names.
```

Comment:
Perhaps expand the docstring for these parameters a bit, or provide an example of usage? From the docstring alone I can't tell how this is intended to be used.

Reply:
I'll add an example.

Reply:
Done.